lib/amazon/collector/ec2.rb

```diff
@@ -2,7 +2,8 @@ module Amazon
   class Collector
     module Ec2
       def source_regions(scope)
-        return [] unless scope[:region] == "us-east-1"
+        # We want to collect this for only default region, since all regions return the same result
+        return [] unless scope[:region] == default_region
```

Comment:
I think we should collect source_regions / flavors outside of the context of the regions instead of calling these methods from every region and just returning `[]` but we can revisit that.